python/rmm/rmm/_lib/memory_resource.pyx

```diff
@@ -1188,3 +1193,12 @@ def get_log_filenames():
         else None
         for i, each_mr in _per_device_mrs.items()
     }
+
+
+def available_device_memory():
+    """
+    Returns a tuple of free and total device memory memory.
```

Comment:
I wish this was in a `Returns` block that follows numpy docstring style but it doesn't seem like RMM uses full docstrings...

Reply:
RMM Python is immature in a lot of ways. No time like the present...